fluid/neural_machine_translation/transformer/README_cn.md

```diff
@@ -154,9 +154,64 @@ perl multi-bleu.perl data/newstest2013.tok.de < predict.tok.txt
 ```
 BLEU = 25.08, 58.3/31.5/19.6/12.6 (BP=0.966, ratio=0.967, hyp_len=61321, ref_len=63412)
 ```
+###分布式训练
```

Comment:
<img width="927" alt="2018-07-10 3 57 16" src="https://user-images.githubusercontent.com/3064195/42496899-109b9fc8-845a-11e8-93e0-0fef6b4bd508.png">

Please  pay attention to the format

Reply:
ok